packages/admin/cms-admin/src/dam/Table/damActions/DamActions.tsx

```diff
@@ -31,8 +36,11 @@ export const DamActions: React.VoidFunctionComponent<DamActionsProps> = ({ files
     const client = useApolloClient();
     const damMultiselectApi = useDamMultiselectApi();
 
+    const [archiveLoading, setArchiveLoading] = React.useState<boolean>(false);
+    const [restoreLoading, setRestoreLoading] = React.useState<boolean>(false);
+
     const [deleteDialogOpen, setDeleteDialogOpen] = React.useState<boolean>(false);
-    const [deleting, setDeleting] = React.useState(false);
+    const [deleteLoading, setDeleteLoading] = React.useState(false);
```

Comment:
I liked `deleting` more. Why not name the other states `archiving` and `restoring`?